src/client/cypress/README.md

```diff
@@ -0,0 +1,56 @@
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/AdaptiveConsulting/ReactiveTraderCloud/CI)](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/actions?query=workflow%3ACI)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/AdaptiveConsulting/ReactiveTraderCloud)](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/releases/latest)
+[![GitHub](https://img.shields.io/github/license/AdaptiveConsulting/ReactiveTraderCloud)](https://opensource.org/licenses/Apache-2.0)
+
+[![image](https://raw.githubusercontent.com/AdaptiveConsulting/ReactiveTrader/master/images/adaptive-logo.png)](http://weareadaptive.com/)
+
+# Reactive Trader®
+
+Reactive Trader® is a real-time FX trading platform designed to showcase reactive programming principles across the full application stack.
+
+Originally [written in WPF and .Net](https://github.com/AdaptiveConsulting/ReactiveTrader), and now in React/Redux, .Net and Node.js, we continue to evolve the platform to use the latest technologies.
+
+Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full list of the latest features.
+
+![image](docs/reactive-trader.gif)
+
+
+## Setting up the Test
+
+
+1. Fork and clone the ReactiveTraderCloud repo
+
+2. Navigate to `/src/client` and install Cypress ([from the Cypress website](https://docs.cypress.io/guides/getting-started/installing-cypress))
```

Comment:
Shouldn't need to do this, cypress should be installed as part of the repo's dependencies

Reply:
fixing the review comments in new commit